openedx/core/djangoapps/content_tagging/rules.py

```diff
@@ -101,5 +235,5 @@ def can_change_taxonomy_tag(user: UserType, tag: oel_tagging.Tag | None = None)
 
 # This perms are used in the tagging rest api from openedx_tagging that is exposed in the CMS. They are overridden here
 # to include Organization and objects permissions.
-rules.set_perm("oel_tagging.change_objecttag_taxonomy", can_change_object_tag_taxonomy)
+rules.set_perm("oel_tagging.change_objecttag_taxonomy", can_view_taxonomy)
```

Comment:
We updated the permission names in oel_tagging for `object_tag` to use `objecttag` instead -- which I can see here, but they're not updated in lines 231-234. I think they need to be?

Reply:
Sure!